Sources/Optimizely+Decide/OptimizelyClient+Decide.swift

```diff
@@ -0,0 +1,41 @@
+/****************************************************************************
```

Comment:
```suggestion
/***************************************************************************
```
Same for all modified or new files.